src/store/lang-store.js

```diff
@@ -0,0 +1,41 @@
+import { app } from '../main'
```

Comment:
Den her synes jeg er lidt træls og contra-intuitiv. Er det virkelig måden at gøre det på? Kan godt se, hvad du vil opnå, men gøres det ved at importere hele main for at få en reference til Vue?

Findes der ikke en lokal reference eller kan man ikke give en reference med? Det er åbne spørgsmål - synes bare det virker uldent at skulle importere Vue på den måde i en store

Reply:
Kan huske vi rodede en del med at få fat på $i18n variablen, og søgte også en del på det. Fik det til at virke med app referencen og beholdte det. Måske vi skal have en separat JIRA på denne forbedring?

Reply:
Ok. Jeg har lavet en jira - https://sbprojects.statsbiblioteket.dk/jira/browse/DIGISAM-130